charts/opensearch/values.yaml

```diff
@@ -392,6 +392,7 @@ ingress:
   annotations: {}
     # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
+  labels: {}
```

Comment:
Hey @wigust thanks for your contribution and patience
did you try with the following already part of the chart for custom labels?
```
# additionals labels
labels: {}
```

If you are looking for custom labels only for ingress, please change to `ingressLabels: {}`

Thank you

Reply:
Hello @prudhvigodithi

Thank you for the review.

> did you try with the following already part of the chart for custom labels?

I didn't, and I see that `labels` will work but it triggers to restart all OpenSearch pods which is too costly for the case of adding a label to ingress.

> If you are looking for custom labels only for ingress, please change to ingressLabels: {}

I replaced the option and tested with:

```shell
helm template -f values.yaml ~/src/github.com/opensearch-project/helm-charts/charts/opensearch
```

```yaml
ingress:
  ingressLabels:
    foo: bar
```